docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+# convenient way to start memcached for specs that need it: docker-compose up -d
+services:
```

Comment:
I'm not yet ready to say this repo needs this file

Reply:
Fair. There isn't much to it, but does give a simple way to manage it. Removed for now and suggest a docker command instead.